src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -351,6 +338,78 @@ private SmugglerProgressBase.Counts ProcessIndexes(SmugglerResult result)
             return result.Indexes;
         }
 
+        private void WriteIndex(SmugglerResult result, IndexDefinition indexDefinition, IIndexActions actions, bool retryOnError)
+        {
+            try
+            {
+                if (_options.RemoveAnalyzers)
+                {
+                    foreach (var indexDefinitionField in indexDefinition.Fields)
+                        indexDefinitionField.Value.Analyzer = null;
+                }
+
+                actions.WriteIndex(indexDefinition);
+            }
+            catch (Exception e)
+            {
+                var exceptionMessage = e.Message;
+                if (exceptionMessage.Contains("No overload for method 'LoadDocument' takes 1 arguments"))
```

Comment:
You are relying on the _error message_ from the compiler.
But if you are using a machine with localized error messages, see:
https://github.com/dotnet/roslyn/blob/ac2da0f6985620a8dea9aca7c2a00848df3b4e8f/src/Compilers/CSharp/Portable/CSharpResources.resx#L666

Reply:
done